app/models/bike.rb

```diff
@@ -1,7 +1,10 @@
 class Bike < ApplicationRecord
-  validates_presence_of    :identifier
-  validates_uniqueness_of  :identifier
-  
-  belongs_to :current_station, class_name: :Station, foreign_key: :current_station_id, optional: true
+    validates_presence_of   :identifier,
+                            :bike_model,
```

Comment:
What is bike_model in terms of our work? Would this show in any relations to the maps?

Reply:
`bike_model` is some filler information. If we had multiple types of bikes or different models of them (i.e. bikes that came out in different years). I think we could honestly take it off because we wouldn't use it unless we actually had bikes and we probably wouldn't be interested in this information from a user standpoint.